packages/features/package.json

```diff
@@ -14,6 +14,7 @@
     "dompurify": "^2.4.1",
     "framer-motion": "^10.12.3",
     "lexical": "^0.5.0",
+    "react-select": "^5.7.3",
```

Comment:
@zomars i was getting react-select errors still. Updated both to 5.7.3 and they disapeared Not sure if this is relevant to what we were discussing the other day